src/include/sof/audio/mux.h

```diff
@@ -37,7 +37,7 @@ STATIC_ASSERT(MUX_MAX_STREAMS < PLATFORM_MAX_STREAMS,
 
 struct mux_stream_data {
 	uint32_t pipeline_id;
-	uint8_t num_channels;
+	uint8_t reserved1;	/* reserved as no longer used */
```

Comment:
these should all have "_deprecated" suffix with a comment on version last supported.

Reply:
Sorry, can you confirm if these were ever used externally ? If true then comment can be removed, if false then we need the "_deprecated" suffix and the comment indicating the ABI version they were deprecated.